draft-ietf-quic-recovery.md

```diff
@@ -430,13 +430,13 @@ thresholds to minimize recovery latency.
 
 The RECOMMENDED initial value for the packet reordering threshold
 (kPacketThreshold) is 3, based on best practices for TCP loss detection
-{{?RFC5681}} {{?RFC6675}}.  Implementations SHOULD NOT use a packet threshold
-less than 3, to keep in line with TCP {{?RFC5681}}.
+({{?RFC5681}}, {{?RFC6675}}).  Implementations SHOULD NOT use a packet threshold
+less than 3, to keep in line with TCP ({{?RFC5681}}).
```

Comment:
"to keep in line with" felt awkward, but I didn't come up with a rephrasing I was confident of.  Maybe "comparable to TCP"?

Reply:
Maybe "In order to remain similar to TCP, implementations SHOULD NOT use a packet threshold less than 3."?